rust/src/rfb/rfb.rs

```diff
@@ -226,7 +245,11 @@ impl RFBState {
                             consumed += current.len() - rem.len();
                             current = rem;
 
-                            self.state = parser::RFBGlobalState::TCSecurityResult;
+                        if let Some(frame) = &_pdu {
+                            frame.set_len(flow, consumed as i64);
+                        }
+
+                        self.state = parser::RFBGlobalState::TCSecurityResult;
```

Comment:
nit: improper indent

Reply:
I was under the impression that it will be fixed after rustfmt